pkg/cloudproviders/aws/cloudprovider/cloudprovider.go

```diff
@@ -56,6 +59,7 @@ const (
 
 func init() {
 	v1alpha5.NormalizedLabels = lo.Assign(v1alpha5.NormalizedLabels, map[string]string{"topology.ebs.csi.aws.com/zone": v1.LabelTopologyZone})
+	utilruntime.Must(awsapis.AddToScheme(scheme.Scheme))
```

Comment:
Should this be at a higher layer than the cloudprovider (maybe in `context`) since other things in `cloudproviders/aws` might use it?

Reply:
Discussed offline. Let's let this evolve.